MSAL/src/instance/MSALAccountsProvider.m

```diff
@@ -441,41 +440,4 @@ - (BOOL)setCurrentPrincipalAccountId:(MSIDAccountIdentifier *)currentAccountId e
                                                                     error:error];
 }
 
-#pragma mark - Completion block
-
-- (BOOL)setCurrentSSOExtensionRequest:(MSIDSSOExtensionGetAccountsRequest *)request API_AVAILABLE(ios(13.0), macos(10.15))
-{
-    @synchronized (self)
-    {
-        if (self.currentRequest)
-        {
-            MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Request is already executing. Please wait or cancel the request before starting it again.");
-            return NO;
-        }
-        
-        self.currentRequest = request;
-        return YES;
-    }
-    
-    return NO;
-}
-
-- (MSIDSSOExtensionGetAccountsRequest *)copyAndClearCurrentSSOExtensionRequest API_AVAILABLE(ios(13.0), macos(10.15))
-{
-    @synchronized (self)
-    {
-        if (!self.currentRequest)
```

Comment:
`currentRequest` is not a static variable, how can it prevent multiple `allAccountsFromDevice` calls from running at the same time?

Another quesiton is: why do we only allow one `allAccountsFromDevice` call at a time? :)

Reply:
@jasoncoolmax, just an optimization. I didn't see a reason why would developer need multiple requests for deviceInfo for the same public client app, but I might be wrong. I thought to have one instance of deviceInfo provider per public client application (need to change current code a bit), and not enforce it globally across process. Not feeling very strong neither ways, so if you have other suggestions, more than happy to adopt them :)